aiorospy/src/aiorospy/action.py

```diff
@@ -139,6 +157,10 @@ def __init__(self, name, action_spec, loop=None):
         self._loop = loop if loop is not None else asyncio.get_event_loop()
         self._client = ActionClient(name, action_spec)
         self._status_sub = AsyncSubscriber(name + "/status", GoalStatusArray, loop=self._loop, queue_size=1)
+        self._exc_q = janus.Queue(loop=loop)
```

Comment:
`_exception_queue`?  No documentation + abbreviated names conspires to make this hard to understand as easily as it should be.